include/ConfigManager.h

```diff
@@ -240,6 +245,14 @@ class LMMS_EXPORT ConfigManager : public QObject
 
 	void addRecentlyOpenedProject(const QString & _file);
 
+	static void addFavoriteItem(QString& item);
+	static void removeFavoriteItem(QString& item);
+	static bool isFavorite(QString& item);
```

Comment:
Maybe `addFavoriteFile`, `removeFavoriteFile`, and `isFavoriteFile`? I also dont think you need to pass in `item` as a simple reference. Surely you don't need the actual `QString` object, but the text it contains, so I would use `const QString&`, or just `QString` since `QString`'s uses implicit sharing, but `const QString&` wouldn't hurt/just to be safe.